gems/sorbet-runtime/lib/types/helpers.rb

```diff
@@ -33,7 +35,7 @@ def sealed!
   #  end
   #
   # Except that it is statically analyzed by sorbet.
-  def mixes_in_class_methods(mod)
-    Private::Mixins.declare_mixes_in_class_methods(self, mod)
+  def mixes_in_class_methods(mod, *mods)
+    Private::Mixins.declare_mixes_in_class_methods(self, Array(mod).concat(mods))
```

Comment:
We can do:
```suggestion
    Private::Mixins.declare_mixes_in_class_methods(self, [mod].concat(mods))
```
to not need `Kernel`. We don't expect `mod` to be an array itself, anyway.